.changeset/dull-gorillas-unite.md

```diff
@@ -1,5 +1,5 @@
 ---
-"@vercel/blob": patch
+"@vercel/blob": minor
```

Comment:
Let's avoid modifying the changelog (but I guess this change will be ignored if we merge https://github.com/vercel/storage/pull/586)